ecc_enclave/enclave/shim.cpp

```diff
@@ -49,7 +49,10 @@ void get_state(const char* key, uint8_t* val, uint32_t max_val_len, uint32_t* va
         sgx_sha256_msg(val, *val_len, &state_hash);
     }
 
-    if (check_cmac(key, NULL, &state_hash, &session_key, &cmac) == 0) {
+    if (check_cmac(key, NULL, &state_hash, &session_key, &cmac) != 0) {
+        LOG_ERROR("Enclave: VIOLATION!!! Oh oh! cmac does not match!");
```

Comment:
can we just remove this for now and add an issue?

Reply:
ok, let's tackle this with #83